tools/issue-labeler/eng/Versions.props

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project>
+  <PropertyGroup>
+    <!-- Libs -->
+    <MicrosoftAspNetCoreAllVersion>2.1.0</MicrosoftAspNetCoreAllVersion>
+    <!-- update MicrosoftDataAnalysisVersion to 2.0.0 -->
+    <MicrosoftDataAnalysisVersion>0.1.0</MicrosoftDataAnalysisVersion>
+    <OctokitVersion>10.0.0</OctokitVersion>
+    <MicrosoftAzureKeyVaultVersion>3.0.5</MicrosoftAzureKeyVaultVersion>
+    <MicrosoftExtensionsVersion>6.0.0</MicrosoftExtensionsVersion>
+    <MicrosoftDotNetGitHubIssueLabelerAssetsVersion>1.12.19</MicrosoftDotNetGitHubIssueLabelerAssetsVersion>
```

Comment:
I don't **think** this applies to us, but we should double check.

Reply:
We use OctoKit in several files, the other three we don't use.